pkg/logging/logger.go

```diff
@@ -95,15 +79,13 @@ func (l Logger) Error(err error, message string, kvpair ...interface{}) {
 			Stack,
 			le.Stack())
 		l.Real.Info(message, kvpair...)
-		l.history[err] = true
-	} else {
-		_, found := l.history[err]
-		if found || errors.IsConflict(err) {
-			return
-		}
-		l.Real.Error(err, message, kvpair...)
-		l.history[err] = true
+		return
+	}
+	err = errors.Unwrap(err)
```

Comment:
do we have to worry at all about the case where `errors.Unwrap(err)` returns nil?

Reply:
Perhaps.  Our `Error` will not wrap a `nil` error but another implementation (which is what this Unwrap() is for) might.  I'll add the check.